Makefile

```diff
@@ -185,28 +184,57 @@ container-test:
 		-t $(DOCKER_REPO):local_e2e_test .
 endif
 
-.PHONY: container
-container: Dockerfile
-	$(OCI_BIN) build --build-arg BUILD_DATE="$(BUILD_TIMESTAMP)" \
+.PHONY: container-build
+container-build:
+	git update-index --refresh
+	$(OCI_BIN) buildx build \
+		--platform linux/amd64,linux/arm64 \
```

Comment:
So when I take a look at the manifests on `douglascamata/obs-api-test` I see certain "unknown on unknown" manifests alongside the regular ones. Is this expected?
![Screenshot from 2023-06-15 09-01-52](https://github.com/observatorium/api/assets/51132453/dbbeb296-277a-4ea0-941c-91ee4e34436c)

I also see them for receive-controller and token-refresher but not for up and rules-objstore images.

Reply:
This is part of the provenance feature included and recent versions of buildx by default: https://docs.docker.com/build/attestations/slsa-provenance/. Quay seems to not support them very well by now.

Reply:
```
$ docker buildx imagetools inspect quay.io/douglascamata/obs-api-test
Name:      quay.io/douglascamata/obs-api-test:latest
MediaType: application/vnd.oci.image.index.v1+json
Digest:    sha256:6b4711c097cc8f8fe55782febfc00d0d4edf3d8cde445e310a30921f3b7ead79
           
Manifests: 
  Name:        quay.io/douglascamata/obs-api-test:latest@sha256:d6c946f7e1b2c243efda345ba7718ad32ae2876167ccfa9bc09242e99355425f
  MediaType:   application/vnd.oci.image.manifest.v1+json
  Platform:    linux/amd64
               
  Name:        quay.io/douglascamata/obs-api-test:latest@sha256:2ab7b84462be9e7637c13025b880af4b1054c895b50bbe9f6c3e4339d0432548
  MediaType:   application/vnd.oci.image.manifest.v1+json
  Platform:    linux/arm64
               
  Name:        quay.io/douglascamata/obs-api-test:latest@sha256:ac7fe4659ae70ed169fade5a8c75e6acc785f2329d1576883cfbf90cd3b3ee67
  MediaType:   application/vnd.oci.image.manifest.v1+json
  Platform:    unknown/unknown
  Annotations: 
    vnd.docker.reference.digest: sha256:d6c946f7e1b2c243efda345ba7718ad32ae2876167ccfa9bc09242e99355425f
    vnd.docker.reference.type:   attestation-manifest
               
  Name:        quay.io/douglascamata/obs-api-test:latest@sha256:323bf88a4a204d87316b9050282bdcc2f85629538072e5ba32c0186893a100dc
  MediaType:   application/vnd.oci.image.manifest.v1+json
  Platform:    unknown/unknown
  Annotations: 
    vnd.docker.reference.digest: sha256:2ab7b84462be9e7637c13025b880af4b1054c895b50bbe9f6c3e4339d0432548
    vnd.docker.reference.type:   attestation-manifest
```